novawallet/Modules/ManualBackupKeyList/ManualBackupKeyListPresenter.swift

```diff
@@ -0,0 +1,200 @@
+import Foundation
+import SoraFoundation
+import RobinHood
+
+final class ManualBackupKeyListPresenter {
+    weak var view: ManualBackupKeyListViewProtocol?
+    let wireframe: ManualBackupKeyListWireframeProtocol
+    let interactor: ManualBackupKeyListInteractorInputProtocol
+
+    private let metaAccount: MetaAccountModel
+    private let walletViewModelFactory = WalletAccountViewModelFactory()
+    private let networkViewModelFactory: NetworkViewModelFactoryProtocol
+    private let localizationManager: LocalizationManagerProtocol
+    private let logger: Logger
+
+    private var chains: [ChainModel.Id: ChainModel] = [:]
+
+    init(
+        interactor: ManualBackupKeyListInteractorInputProtocol,
+        wireframe: ManualBackupKeyListWireframeProtocol,
+        metaAccount: MetaAccountModel,
+        networkViewModelFactory: NetworkViewModelFactoryProtocol,
+        localizationManager: LocalizationManagerProtocol,
+        logger: Logger
+    ) {
+        self.interactor = interactor
+        self.wireframe = wireframe
+        self.metaAccount = metaAccount
+        self.networkViewModelFactory = networkViewModelFactory
+        self.logger = logger
+        self.localizationManager = localizationManager
+    }
+}
+
+// MARK: ManualBackupKeyListPresenterProtocol
+
+extension ManualBackupKeyListPresenter: ManualBackupKeyListPresenterProtocol {
+    func setup() {
+        interactor.setup()
+
+        guard let walletViewModel = try? walletViewModelFactory.createDisplayViewModel(from: metaAccount) else {
+            return
+        }
+
+        view?.updateNavbar(with: walletViewModel)
+    }
+
+    func didTapDefaultKey() {
```

Comment:
We usually use ```did``` prefix for output protocols. For presenter we need to perform action, so let's rename to ``activateDefaultKey``` and ```activateCustomKey```